src/test/java/pl/gov/coi/pomocua/ads/myoffers/MyOffersResourceTest.java

```diff
@@ -107,4 +122,10 @@ void shouldReturnNotFoundWhenGettingOfferForOtherUser() {
         assertThat(entity).usingRecursiveComparison().ignoringFields("id").isEqualTo(request);
         return entity;
     }
+
+    private <T extends BaseOffer> void deleteOffer(String urlSuffix, Class<T> clazz, Long id) {
+        ParameterizedTypeReference<AccommodationsResource> accommodationsResource = new ParameterizedTypeReference<AccommodationsResource>() {};
+        ResponseEntity<AccommodationsResource> responseDelete = restTemplate.exchange("/api/secure/" + urlSuffix + "/" + id, HttpMethod.DELETE, null, accommodationsResource, clazz);
```

Comment:
It could be much simplified to:
`restTemplate.exchange("...url...", HttpMethod.DELETE, null, Void.class)`